### PR TITLE
Add dashboard for sponsors

### PR DESCRIPTION
**Pull request by zoek1**
##### Description

https://www.loom.com/share/060db786639a4ca2bfaa126503fd7e63
image

##### Refers/Fixes
https://github.com/gitcoinco/web/issues/5669

### CODE REVIEW

**Comment by frankchen07** [comment]
some comments:

1. I'm not a huge fan of putting a button for the sponsor hack dashboard here, but I guess that's something we can change later on.
Screen Shot 2020-07-21 at 12 25 18

2. What is this from?
Screen Shot 2020-07-21 at 12 25 29

3. There's ALOT of information on this page. The density is pretty high. I would move the "view prize" as just a clickable prize name -> takes them to the prize if they desire.
Screen Shot 2020-07-21 at 12 25 48

4. Again, awesome to see this happening, there's just alot of text on the page, and there's probably a way to make it stand out better.
Screen Shot 2020-07-21 at 12 26 17

5. I don't know where this goes, but there has been conversation about how "finding more hackers" is the value of our network and our product. Sponsors shouldn't even have to touch "finding people", unless it is a means to having a conversation with someone and creating a relationship. That's a different "finding" action.
Screen Shot 2020-07-21 at 12 26 40

**Comment by zoek1** [comment]
2. it was deleted in most recent update
3. I made the changes for this in most recent version
4. This was implemented request by @PixelantDesign yesterday
image
5. Goes to the user directoy

**Comment by PixelantDesign** [comment]
last change @zoek1 could we make the switch blue color - same as chat icon color?

File: app/dashboard/templates/dashboard/sponsors.html [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+                                <span v-if="!isMobileDevice && prize.value_eth.toFixed" class="tag token ml-2">[[prize.value_eth.toFixed(2)]] ETH</span>
+                                <span v-if="!isMobileDevice && prize.values_usdt" class="tag usd">[[prize.values_usdt]] USD</span>
+                              </div>
+                              <span v-if="isMobileDevice" class="font-body font-weight-semibold">[[ prize.total_submitted ]]/[[prize.total_projects]] Submitted</span>
+                            </div>
+                            <div>
+                              <span v-if="!isMobileDevice" class="font-body font-weight-semibold">[[ prize.total_submitted ]]/[[prize.total_projects]] Submitted</span>
+                              <span v-if="prize.paid" class="ml-5 font-body font-weight-semibold gc-text-green"><span class="fas fa-check"></span>  Paid</span>
+                              <a v-if="!prize.paid" :href="`/issue/payout?pk=${prize.pk}`" class="ml-5 font-body font-weight-semibold">Unpaid</a>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/sponsors.html:175]: @zoek1 @PixelantDesign This is where it was added in!

**Pull request merged**